boards/arm/qomu/Kconfig.defconfig

```diff
@@ -0,0 +1,20 @@
+# QuickLogic Qomu board
+
+# Copyright (c) 2022 Antmicro <www.antmicro.com>
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_QOMU
+
+config BOARD
+	default "qomu"
+
+config UART_PL011
+	default n
+
+config UART_INTERRUPT_DRIVEN
+	default n
+
+config UART_USBSERIAL
```

Comment:
remove this, should be determined by devicetree having the node enabled/disabled.